### PR TITLE
 Add example for sending notification to a reply in message in channel

### PR DESCRIPTION
**Pull request by mehakagarwal**

### CODE REVIEW

File: api-reference/beta/api/team-sendactivitynotification.md [hunk 1]
```diff
@@ -555,6 +555,65 @@ Content-Type: application/json
...
+  "blockType": "response",
+  "truncated": false
+}
+-->
+``` http
+HTTP/1.1 204 No Content
+
+```
+
+### Example 7: Notify about pending finance approval requests in channel Message Reply Location
```
Comment (eddie-lee-msft) [inline api-reference/beta/api/team-sendactivitynotification.md:568]: please lower-case the reply location

**Review by eddie-lee-msft — APPROVED** [review]

File: api-reference/beta/api/team-sendactivitynotification.md [hunk 2]
```diff
@@ -555,6 +555,64 @@ Content-Type: application/json
...
+  "blockType": "response",
+  "truncated": false
+}
+-->
+``` http
+HTTP/1.1 204 No Content
+
+```
+
+### Example 7: Notify about pending finance approval requests in channel Message reply location
```
Comment (Danielabom) [inline api-reference/beta/api/team-sendactivitynotification.md:568]:
```suggestion
### Example 7: Notify about pending finance approval requests in channel message reply location
```

File: api-reference/v1.0/api/team-sendactivitynotification.md [hunk 3]
```diff
@@ -589,6 +589,63 @@ The following example shows the response.
 HTTP/1.1 204 No Content
 ```
 
+### Example 7: Notify about pending finance approval requests in channel Message reply location
```
Comment (Danielabom) [inline api-reference/v1.0/api/team-sendactivitynotification.md:592]:
```suggestion
### Example 7: Notify about pending finance approval requests in channel message reply location
```

**Review by Danielabom — APPROVED** [review]
PR approved.

**Pull request merged**